src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = date.split(fromFormat[3]);
+  const formated = [];
+  const indexYearShort = fromFormat.indexOf('YY');
+
+  if (fromFormat.includes('YY') && newDate[indexYearShort] >= 30) {
+    newDate[indexYearShort] = '19' + newDate[indexYearShort];
+  }
+
+  if (fromFormat.includes('YY') && newDate[indexYearShort] < 30) {
+    newDate[indexYearShort] = '20' + newDate[indexYearShort];
+  }
+
+  fromFormat[indexYearShort] = 'YYYY';
```

Comment:
Better to make constants for all hardcoded strings  and numbers outside of function.
Esprcially which uses more than one time.

Reply:
I didn't understand. Should I move all of variables outside of function? or whats wrong with code on lines 62-65?